sepal_ui/mapping/mapping.py

```diff
@@ -782,3 +784,20 @@ def get_viz_params(image):
                     props = {}
 
         return props
+
+    def add_layer(self, layer, hover=False):
+        """Add layer and use a default style for the GeoJSON inputs
+
+        default_hover (bool): whether to use the default hover style or not.
+
+        """
+
+        if isinstance(layer, GeoJSON):
+
+            if not layer.style:
+                layer.style = styles.layer_style
```

Comment:
for coverage purpose, could you add a single test, with a fake geoJson (like a triangle anywhere) so that these 2 if statement are used at least once in the test suit